yaml-tests/src/test/resources/enum.yamsql

```diff
@@ -0,0 +1,112 @@
+          VALUES (14, 'alice'),
+                 (15, 'bob')
+---
+test_block:
+  tests:
+    -
+      - query: SELECT * From TABLE_A where "mood" = 'CONFUSED'
+      - unorderedResult: [{2, 'bar', 'CONFUSED'},
+                          {11, 'plugh', 'CONFUSED'}]
+    -
+      - query: SELECT * From TABLE_A where "mood" > 'INDIFFERENT'
+      - unorderedResult: [{2, 'bar', 'CONFUSED'},
+                          {4, 'qux', 'ANGRY'},
+                          {7, 'grault', 'ANXIOUS'},
+                          {8, 'garply', 'SAD'},
+                          {11, 'plugh', 'CONFUSED'},
+                          {13, 'thud', 'SAD'}]
+    -
+      - query: SELECT * From TABLE_A where "mood" >= 'INDIFFERENT'
+      - unorderedResult: [{2, 'bar', 'CONFUSED'},
+                          {4, 'qux', 'ANGRY'},
+                          {7, 'grault', 'ANXIOUS'},
+                          {8, 'garply', 'SAD'},
+                          {10, 'fred', 'INDIFFERENT'},
+                          {11, 'plugh', 'CONFUSED'},
+                          {12, 'xyzzy', 'INDIFFERENT'},
+                          {13, 'thud', 'SAD'}]
+    -
+      - query: SELECT * From TABLE_A where "mood" < 'INDIFFERENT'
+      - unorderedResult: [{1, 'foo', 'HAPPY'},
+                          {3, 'baz', 'JOYFUL'},
+                          {5, 'quux', 'RELAXED'},
+                          {6, 'corge', 'JOYFUL'},
+                          {9, 'waldo', 'JOYFUL'}]
+    -
+      - query: SELECT * From TABLE_A where "mood" <= 'INDIFFERENT'
+      - unorderedResult: [{1, 'foo', 'HAPPY'},
+                          {3, 'baz', 'JOYFUL'},
+                          {5, 'quux', 'RELAXED'},
+                          {6, 'corge', 'JOYFUL'},
+                          {9, 'waldo', 'JOYFUL'},
+                          {10, 'fred', 'INDIFFERENT'},
+                          {12, 'xyzzy', 'INDIFFERENT'}]
+    -
+      - query: SELECT * From TABLE_A where "mood" != 'INDIFFERENT'
+      - unorderedResult: [{1, 'foo', 'HAPPY'},
+                          {2, 'bar', 'CONFUSED'},
+                          {3, 'baz', 'JOYFUL'},
+                          {4, 'qux', 'ANGRY'},
+                          {5, 'quux', 'RELAXED'},
+                          {6, 'corge', 'JOYFUL'},
+                          {7, 'grault', 'ANXIOUS'},
+                          {8, 'garply', 'SAD'},
+                          {9, 'waldo', 'JOYFUL'},
+                          {11, 'plugh', 'CONFUSED'},
+                          {13, 'thud', 'SAD'}]
+    -
+      - query: SELECT * From TABLE_A where "mood" IS NOT NULL AND "id" < 5
+      - unorderedResult: [{1, 'foo', 'HAPPY'},
+                          {2, 'bar', 'CONFUSED'},
+                          {3, 'baz', 'JOYFUL'},
+                          {4, 'qux', 'ANGRY'}]
+    -
+      - query: SELECT * From TABLE_A where "mood" IS NULL
+      - unorderedResult: [{14, 'alice', !null },
+                          {15, 'bob', !null }]
+    -
```

Comment:
Can you add an inequality with a string, i.e. "mood" < "something"

Reply:
done!